DOCS-409 Security index (#1032)
neo4j/docs-aura: modules/ROOT/pages/security/index.adoc

= Security
:description: This section covers important security aspects in Neo4j.

> This section of documentation describes how to ensure security according to industry best practices.

Reviewer: ```suggestion
This section describes how to ensure security of your Neo4j Aura instances.
```

Neo4j Aura is available on multiple commercial cloud service
providers and in multiple regions.

> As with any modern cloud service, Neo4j Aura, the provider and customers share responsibility for using the service securely.

Reviewer: ```suggestion
The responsibility for using the service securely is shared between Neo4j Aura, the cloud provider, and the user.
```

* The foundational layer is the cloud infrastructure that the service runs on. 
By utilizing cloud infrastructure providers, Neo4j Aura can rely on uptime and reliability in the building blocks that make up the infrastructure.
* The middle layer is the Neo4j Aura service itself, delivering graph database services to customers around the world. 

> * The top layer is the customer, including user interaction and configuration. 

Reviewer: This needs to be rewritten. It sounds too much like a marketing text.
The foundational layer of what? Security? 
This section is not about third-party cloud infrastructure in isolation, is it? It is about Aura and the measures to take within that. Sometimes it touches upon the cloud providers, but never in isolation. So I think it is wrong to frame it like the foundational layer is not us. 
And remember that our users are users, never customers in terms of documentation.
